app/templates/components/nav-bar.hbs

```diff
@@ -19,10 +19,8 @@
       {{/if}}
       {{#if session.isAuthenticated}}
         {{#ui-dropdown class='item' action='hide'}}
-          <img class="ui avatar round image nav-bar-image"
-               alt="{{authManager.currentUser.email}}"
-               src="{{if authManager.currentUser.avatarUrl authManager.currentUser.avatarUrl
-                         '/images/placeholders/avatar.png'}}">
+          {{widgets/safe-image class='ui avatar round image nav-bar-image' src=(if authManager.currentUser.avatarUrl authManager.currentUser.avatarUrl
```

Comment:
@geekychaser This change has already been made, seems like your branch is out of date. Can you manually rebase the branch?

Reply:
The changes related to this were made in https://github.com/fossasia/open-event-frontend/pull/814

Reply:
sir rebased the branch and commited. Please review it.

thank you